README.md

```diff
@@ -1,24 +1,124 @@
-# README
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Doctors Booking App
 
-Things you may want to cover:
+This is a Doctors online platform for booking appointments.
 
-* Ruby version
+- Menu
+  The Docors section displays a list of all available Doctors. Users can book a doctor by clicking the reservation button.
 
-* System dependencies
+- My appointments
+  The Appointments section displays a list of current appointments along with their brief description.
 
-* Configuration
```

Comment:
Please add a link to a deployed version of your project, It will be a great help for your evaluator. 
I was trying to do a quick QA on my local machine but I had issues with setting up the project (which could be on my side but unfortunately I was not able to solve it quickly enough).

If you manage to add a live demo link - please ask for another review, so we can support you with some QA.